docker-compose.yml

```diff
@@ -9,19 +9,19 @@ services:
         dockerfile: disco/Dockerfile
       command: run_arches
       volumes:
-        - ../arches/:/web_root/arches
+        - ${ARCHES_CORE_DIR:-../arches/}:/web_root/arches
```

Comment:
Is there a way to do this by reusing ARCHES_ROOT, AFS_ROOT, and APP_ROOT so that we have 3 vars and not 6? Just wondering.

Reply:
The difference here is build vs run.  Those variables are build variables set in the Dockerfile.  I'll see about inheriting these variables in the dockerfile - I think you can do it with compose with args - but it's not accurate to say there are 6 variables at play - you'd still have only 3 on the container itself.

Reply:
Yeah, a couple more minutes looking at this - this doesn't make sense for this PR.  The ARCHES_ROOT, AFS_ROOT, APP_ROOT directories are talking about directories _within_ the container.  The other variables (ARCHES_CORE_DIR) are talking about directories _on the docker host_.  So this should not be changed.

Reply:
Thanks for explaining, that's helpful.